test/SignerBondsUniswapV2.test.js

```diff
@@ -74,6 +74,181 @@ describe("SignerBondsUniswapV2", () => {
     })
   })
 
+  describe("setMaxAllowedPriceImpact", () => {
+    const maxAllowedPriceImpact = 10000
+
+    context("when caller is not the governance", () => {
+      it("should revert", async () => {
+        await expect(
+          signerBondsUniswapV2
+            .connect(thirdParty)
+            .setMaxAllowedPriceImpact(maxAllowedPriceImpact)
+        ).to.be.revertedWith("Ownable: caller is not the owner")
+      })
+    })
+
+    context("when max allowed price impact too big", () => {
+      it("should revert", async () => {
+        await expect(
+          signerBondsUniswapV2
+            .connect(governance)
+            .setMaxAllowedPriceImpact(maxAllowedPriceImpact + 1)
+        ).to.be.revertedWith("Maximum value is 10000 basis points")
+      })
+    })
+
+    context("when governance did not set max allowed price impact", () => {
+      const defaultPriceImpact = 100
+      it("should return the default value", async () => {
```

Comment:
This one `describe`s `maxAllowedPriceImpact`, not `setMaxAllowedPriceImpact`. Should we move it to a separate `describe`?

Reply:
Yes, done in https://github.com/keep-network/coverage-pools/pull/156/commits/5f635a73cfd5eebb12950e447acdec55b28b0cd5.